tl

```diff
@@ -538,7 +678,9 @@ if cmd == "run" then
    end
 
    tl.loader()
-   return chunk((unpack or table.unpack)(arg))
+   local res = chunk((unpack or table.unpack)(arg))
```

Comment:
Now that we're doing things after running the script, it might be worth it to pcall the script or wrap it in a coroutine just to guarantee the generated stuff gets cleaned up